.mailmap

```diff
@@ -498,3 +498,4 @@ Zhi Zhang <willzzhang@tencent.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangz.david@outlook.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangzhi@localhost.localdomain>
 Zhuang Xiaochun <zhuangxc89@163.com>
+Zhang Lei <243290414@qq.com> <zhanglei@trendytech.com.cn> 
```

Comment:
reverse the order of emails, the mailmap should follow the preferred corrected syntax